config.go

```diff
@@ -36,22 +37,34 @@ const (
 
 	// ConfigLocation location of the dataset
 	ConfigLocation = "datasetLocation"
+
+	// ConfigPollingTime time after which polling should be done
+	ConfigPollingTime = "pollingTime"
+
+	// ConfigOrderBy lets user decide
+	ConfigOrderBy = "orderby"
```

Comment:
Let's not forget updating the specs (if not already). That can be in the last stage though.

Reply:
Yes. That's in place